tests/components/dyson/test_sensor.py

```diff
@@ -269,7 +303,47 @@ async def test_purecool_component_setup_only_once(devices, login, hass):
     config = _get_config()
     await async_setup_component(hass, dyson_parent.DOMAIN, config)
     await hass.async_block_till_done()
-    discovery.load_platform(hass, "sensor", dyson_parent.DOMAIN, {}, config)
```

Comment:
This call was probably an artificial way of forcing another platform set up to verify that duplicate entities is not created. Not sure if we really need to test that, since it can't happen in the code under test.

Reply:
Restored, just to avoid making more changes that I meant to.